src/components/Map/provider.js

```diff
@@ -0,0 +1,26 @@
+import {get} from 'api';
+
+class OpenFormsProvider {
+  constructor(baseUrl) {
+    this.baseUrl = baseUrl;
+  }
+
+  parse(results) {
+    return results.map(location => ({
+      x: location.latLng.lng,
+      y: location.latLng.lat,
+    }));
+  }
+
+  async search({query}) {
+    const searchUrl = `${this.baseUrl}formio/mapsearch`;
```

Comment:
you can lift this up into the `constructor`, now you're interpolating the API endpoint for every request, while this is only needed once.